### ISSUE TITLE
the operation keys doesn't matches the original design

### ISSUE THREAD

**Issue by stavbsh** [Webiks/CesiumFPS#38]
**what to do:**
change the operation keys and update  the "how to play" screen as the orders below -
Double click on a character's marker to get into its first person view
TAB - switch between semi fpv, fpv and map view modes
W - walk straight forward
R- run straight forward
C - enter\ exit crawling mode
E - enter\ exit a building
S - enter\ exit shooting mode
Space key or left click - Shoot
Arrow keys and mouse pointing - Look around

**Comment by OmKoren** [comment]
Mouse currently has two modes: in (when the cursor is not shown) and out (cursor is shown).
In "out" mode - mouse movements should not move or change characters line of sight, as this mode is meant for clicking on external buttons.

**Comment by OmKoren** [comment]
w = walk
s = backwards
a = left movement
d = right movement
tab = semi fpv, fpv, map view
e = enter/exit building
c = crawling
space = enter 
arrow keys/mouse = look around
enter/left mouse click = shoot
left shift = run

**Comment by stavbsh** [comment]
Fix the operation key for running  to W + Shift

**Comment by davidyaha** [comment]
@stavbsh It is W + shift, you mean in the instructions? 
Most games will just tell you `Shift` to run as you would normally have all movement in all directions

**Comment by stavbsh** [comment]
I meant the instructions. Since you can only move straight (and not in all 4 directions) it makes sense to me to write Shift+W

**Comment by davidyaha** [comment]
Yes but it is an optional feature.. Will talk about it face to face

**Comment by OmKoren** [comment]
in the instruction change running to : "Run Forward     Shift+W"
